index.js

```diff
@@ -22,7 +26,12 @@ function getPercentageScore(score) {}
  * @param {object} score2 - An object representing another score of an assignment. Has the same shape as `score1`.
  * @returns {number} A rounded integer, representing the highest percentage scored among the two given scores.
  */
-function getHighestOfTwoScores(score1, score2) {}
+function getHighestOfTwoScores(score1, score2) {
+  const scoreOne = Math.round(score1.received/score1.max * 100);
```

Comment:
You've already used this logic in the previous function... how could you leverage that?